cattle/plugins/docker/storage.py

```diff
@@ -129,7 +129,8 @@ def _do_image_activate(self, image, storage_pool, progress):
             temp = 'index.' + data.qualifiedName
         if progress is None:
             result = client.pull(repository=temp,
-                                 tag=data.tag, auth_config=auth_config)
+                                 tag=data.tag, auth_config=auth_config,
+                                 insecure_registry=True)
```

Comment:
@hibooboo2 Can you add a comment succinctly explaining why we  are doing this? Explain that the flag only impacts a pre-emptive ping and that it causes problems with self-signed certs on insecure registries...or something like that.